### DISCUSSION TITLE
V3 : Builder how to create a list?

### DISCUSSION THREAD

**Discussion by devhoussam1998** [filamentphp/filament#7990] [Help] [answered]
<img width="205" alt="Screenshot 2023-08-22 at 12 59 26" src="https://github.com/filamentphp/filament/assets/121255432/f3969aaa-e313-4459-bdc4-29311ef0c222">

How to create a list same structure above with filamentphp builder feature?

**Comment by ArtMin96** [comment, marked as answer]
Something like this if you mean form

```php
Builder::make('menu')
    ->blocks([
        Builder\Block::make('heading')
            ->schema([
                TextInput::make('title')
                    ->required(),
                TextInput::make('url')
                    ->required(),
            ]),
    ])
```
